From: Debian Science Maintainers
 <debian-science-maintainers@lists.alioth.debian.org>
Date: Thu, 14 Sep 2023 11:55:38 +0000
Subject: Do not use postfix d in generated library

Forwarded: no
Last-Update: 2021-12-19

Upstream assumed a debug library with d postfix if no NDEBUG flag is present.
Last-Update: 2021-12-19
---
 unittests/integration/test_IkFast.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/unittests/integration/test_IkFast.cpp b/unittests/integration/test_IkFast.cpp
index 943d760..fdd16d5 100644
--- a/unittests/integration/test_IkFast.cpp
+++ b/unittests/integration/test_IkFast.cpp
@@ -132,9 +132,6 @@ TEST(IkFast, LoadWamArmIk)
   ik->setHierarchyLevel(1);
   std::stringstream ss;
   ss << DART_SHARED_LIB_PREFIX << "GeneratedWamIkFast";
-#if (DART_OS_LINUX || DART_OS_MACOS) && !NDEBUG
-  ss << "d";
-#endif
   ss << "." << DART_SHARED_LIB_EXTENSION;
   std::string libName = ss.str();
   std::vector<std::size_t> ikFastDofs{0, 1, 3, 4, 5, 6};
